src/WikisourceApi.php

```diff
@@ -126,6 +135,9 @@ public function fetchWikisources( $cacheLifetime = null ) {
 			$ws = new Wikisource( $this, $this->logger );
 			$ws->setLanguageCode( $langInfo['langCode'] );
 			$ws->setLanguageName( $langInfo['langName'] );
+			if ( $langInfo['langCode'] == "mul" ) {
```

Comment:
Rather than handling this here, you could edit the Sparql (see my comment above).